blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/DiscoveryIO.java

```diff
@@ -3,12 +3,11 @@
 import com.rackspacecloud.blueflood.types.IMetric;
 import java.util.List;
 
-public interface DiscoveryIO {
+public interface DiscoveryIO extends BaseDiscoveryIO {
```

Comment:
I am keeping the old way of getting tokens, which is using 'aggregated ES queries'. If someone decides to not create  separate index for token search, they can still use the old way of doing thing using only one index.